skbio/diversity/__init__.py

```diff
@@ -16,6 +16,233 @@
    alpha
    beta
 
+Functions
+---------
+
+.. autosummary::
+   :toctree: generated/
+
+    alpha_diversity
+    beta_diversity
+    get_alpha_diversity_metrics
+    get_beta_diversity_metrics
+
+Examples
+--------
+
+Create a table containing 7 OTUs and 6 samples:
+
+.. plot::
+   :context:
+
+>>> import numpy as np
+>>> data = [[23, 64, 14, 0, 0, 3, 1],
```

Comment:
I *think* (but am not sure) that we talked about this in SD and decided that we should go with this orientation when we port to skbio, but that's a topic for an RFC. Basically, everyone else uses this orientation (e.g., scipy, sklearn, vegan, ...) so in the spirit of simpler integration with what others are doing, we should use this orientation (but we'll discuss in the RFC).